sample/build.gradle.kts

```diff
@@ -6,12 +6,16 @@ plugins {
 
 kotlin {
     android()
-    js().browser()
+    js {
+        browser()
+        binaries.executable()
```

Comment:
The [documentation](https://kotlinlang.org/docs/js-project-setup.html#execution-environments) never really made this very clear to me. What does `binaries.executable()` give us?

Reply:
Whether or not actual JS is generated for this artifact at this level. `sample` needs it or else it wont run in the browser via `jsRun`. If it's excluded (like it is on the other modules), only the klib is generated and it's the responsibility of a downstream kotlin consumer to generate the JS.

Reply:
Thank you, way better explanation that the official docs. ❤️